requirements.txt

```diff
@@ -19,6 +19,8 @@ gwdatafind>=1.1.3
 
 # Requirements for full pegasus env
 pegasus-wms.api >= 5.0.3
+pyyaml
```

Comment:
I see why gitpython is needed (and I have hit this before, but assumed it was a 'me' problem), but not clear why pyyaml is?

I won't hold this up though

Reply:
Pyyaml is explicitly required in pegasus-wms.common which is a requirement of pegasus-wms.api, so I think that part can be removed.
I'd also add an explicit comment in the requirements to note why this is being added and aid it's removal later.

Reply:
Ah, I know that pyyaml is present in the repo. but wanted to avoid a case where it isn't there. If it is declared in `pegasus-wms.common` then this can be removed, which I've done. ... Also added the requested explicit comment.